proto/qdrant/cloud/cluster/v1/cluster.proto

```diff
@@ -425,27 +425,33 @@ enum DatabaseConfigurationLogLevel {
   DATABASE_CONFIGURATION_LOG_LEVEL_OFF = 6;
 }
 
-// Configuration to setup a Qdrant database in a hybrid cloud.
-// All settings apply to hybrid cloud only.
+// Configuration to setup a Qdrant database.
+// The settings apply to managed and/or hybrid cloud, see documentation on each message for more details.
 message DatabaseConfiguration {
   // The default Qdrant database collection configuration
+  // This setting is for both managed and hybrid cloud clusters, see sub-messages for more details.
```

Comment:
nitpicking: sometimes this comment is added below the "This is an optional field" and sometimes above. It would be great to maintain the same order in all them.